openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/BulkEditEntity.spec.ts

```diff
@@ -588,11 +587,14 @@ test.describe('Bulk Edit Entity', () => {
         .locator('.inovua-react-toolkit-load-mask__background-layer')
         .waitFor({ state: 'detached' });
 
+      await updateButtonResponse;
       await page.waitForSelector('.message-banner-wrapper', {
         state: 'detached',
       });
-      await updateButtonResponse;
-      await toastNotification(page, /details updated successfully/);
+      await toastNotification(
+        page,
+        new RegExp('.*(details updated successfully|Import is in progress).*')
+      );
```

Comment:
Don't think we need to move if we got Import is progress message, since on the next step we are validating the changes please revert this change